src/plugins/interactive_setup/server/routes/enrollment/validate_token.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+import type { RouteDefinitionParams } from '../';
+
+/**
+ * Defines routes to parse and validate Elasticsearch enrollment token.
+ */
+export function defineValidateTokenRoutes({ router }: RouteDefinitionParams) {
```

Comment:
__note:__ not sure if we need this endpoint, I can remove it if we're both not sure yet.

Reply:
I don't think we need this. There's nothing sensitive about the token format so we can decode / validate it on the client. 

You could change this to `internal/interactive_setup/configure` though so we've got the skeleton for the manual configuration endpoints :)

They're all just single routes though (RPC style) so no need to nest them in folders like REST endpoints.

Reply:
> They're all just single routes though (RPC style) so no need to nest them in folders like REST endpoints.

That was logical grouping, not REST one :slightly_smiling_face: I'll remove this for now then.

Reply:
Done.